e2e-tests/src/test/resources/spec-setup/batch-config/batch-customizations.conf

```diff
@@ -1,21 +1,27 @@
 scenarioTypes {
-  "batch": {
-    deploymentConfig: {
-      type: "flinkStreaming"
-      restUrl: ${?FLINK_REST_URL}
+  "batch": ${baseBatchConfig}
+  "batch-with-test-on-real-data": ${baseBatchConfig} {
+    modelConfig.components.flinkTable.useRealDataForTests: true
+    category: "BatchTestOnRealData"
```

Comment:
To have a Designer for e2e tests with testing on real data and on generated data, I had to make another scenario type with another category because of scenario type limitations.

Reply:
It's fine but let's use live mode in the default batch scenario type and random in the new one (see the comment above)